src/utilities/getNumericalTarget.ts

```diff
@@ -0,0 +1,30 @@
+import max from 'ml-array-max';
+import min from 'ml-array-min';
+
+import { ClassType } from '../../types/Class';
+
+/**
+ * Returns the array of metadata numerically categorized.
+ * @param {Array} target Array containing the categories.
+ * @param {Array} test Array containing the result of prediction.
+ * @param {Object} pair Object with information about two classes.
+ * @return {Array<number>} Array containing the categories assinged as numbers.
+ */
+
+export function getNumericalTarget(
+  target: string[],
```

Comment:
All those variable names should be plural, actually this is a general rule in the full project. If an array, plural. So `targets`, `tests`, `pairs`

NB: `data` is plural. In loops you can do `for (const datum of data) {}` (use `datum` for the singular form)